### ISSUE TITLE
feature request: "starts with"

### ISSUE THREAD

**Issue by ConorOBrien-Foxx** [Adriandmen/OEISseeker#1]
This would allow you to search only sequences that start with your given term.

**Comment by Adriandmen** [comment]
Hmm, I'm not sure if I understand you. Can you give an example? :3

**Comment by ConorOBrien-Foxx** [comment]
Of course! Say you want to look for a sequence that starts with "3, 5, 7, 9, ...", so it would eliminate sequences such as "1, 3, 5, 7, 9..."

**Comment by Adriandmen** [comment]
I think that should work with the input `3,5,7,9`. It searches for those specific terms on those specific locations.
